docs/cookbook/platform-sh.rst

```diff
@@ -117,18 +122,15 @@ This is how this file should look like for Sylius:
             rm web/app_test.php
             rm web/app_test_cached.php
             rm -rf var/cache/*
-            php bin/console --env=prod --no-debug --ansi cache:clear
+            php bin/console --env=prod --no-debug --ansi cache:clear --no-warmup
+            php bin/console --env=prod --no-debug --ansi cache:warmup
             php bin/console --env=prod --no-debug --ansi assets:install
-            npm install
-            npm run gulp
+            php bin/console --env=prod --no-debug --ansi theme:assets:install
```

Comment:
Shouldn't it be optional only for users of theme bundle? We can just mention about it

Reply:
Yes, can make a comment on it indeed, it's kind of optional. Actually I don't know what happens if you don't use a theme if you run this command..